astrapy/db.py

```diff
@@ -979,3 +1708,247 @@ def truncate_collection(self, collection_name: str) -> AstraDBCollection:
             collection_name,
             options=existing_collection.get("options"),
         )
+
+
+class AsyncAstraDB:
+    def __init__(
+        self,
+        token: Optional[str] = None,
+        api_endpoint: Optional[str] = None,
+        api_path: Optional[str] = None,
+        api_version: Optional[str] = None,
+        namespace: Optional[str] = None,
+    ) -> None:
+        """
+        Initialize an Astra DB instance.
+        Args:
+            token (str, optional): Authentication token for Astra DB.
+            api_endpoint (str, optional): API endpoint URL.
+            namespace (str, optional): Namespace for the database.
+        """
+        self.client = httpx.AsyncClient()
```

Comment:
We can't make it global as the AsyncClient pools connections and so must always be used from the same event loop.